src/python_testing/TC_DeviceBasicComposition.py

```diff
@@ -338,6 +339,38 @@ class RequiredMandatoryAttribute:
                         success = False
                         continue
 
+        # Validate there are attributes in the global range that are not in the required list
+        allowed_globals = [a.id for a in ATTRIBUTES_TO_CHECK]
+        # also allow event list because it's not disallowed
+        allowed_globals.append(0xFFFA)
```

Comment:
Can we use a symbolic name for 0xFFFA to make it clear beyond the comment as well

Reply:
sure - will add.

Reply:
added